[fx] add test for args/kwargs handling

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43575 [fx] avoid naming collisions with built-in names
* **#43640 [fx] add test for args/kwargs handling**

+ added a `self.checkGraphModule` utility function to wrap the common
test assert pattern.

Differential Revision: [D23356262](https://our.internmc.facebook.com/intern/diff/D23356262)

suo committed:
[fx] add test for args/kwargs handling

+ added a `self.checkGraphModule` utility function to wrap the common
test assert pattern.

[ghstack-poisoned]

suo committed:
Update on "[fx] add test for args/kwargs handling"


+ added a `self.checkGraphModule` utility function to wrap the common
test assert pattern.

[ghstack-poisoned]

jamesr66a commented on test/test_fx.py:
```diff
@@ -15,6 +15,21 @@
+        """Check that an nn.Module's results match the GraphModule version
+        for a given set of args/kwargs.
+        """
+        if kwargs is not None:
```
Can we just do `kwargs = kwargs if kwargs else {}` above?

suo replied:
sure, #python

jamesr66a commented on test/test_fx.py:
```diff
@@ -55,7 +70,16 @@ def forward(self, A, b=4, *args, c=5, **kwargs):
 
         t = T()
-        symbolic_trace(t)
+        s = symbolic_trace(t)
```
Is this value `s` supposed to be used somewhere?

suo replied:
oops, left this in

jamesr66a reviewed (approved):
![image](https://user-images.githubusercontent.com/4685384/91354334-73fe0680-e7a1-11ea-9624-3646c9cfb720.png)

suo committed:
Update on "[fx] add test for args/kwargs handling"


+ added a `self.checkGraphModule` utility function to wrap the common
test assert pattern.

Differential Revision: [D23356262](https://our.internmc.facebook.com/intern/diff/D23356262)

[ghstack-poisoned]
